### DISCUSSION TITLE
Import of Neworks and Merchants do not work

### DISCUSSION THREAD

**Discussion by fiversen-hofor** [datafeedr/wordpress-plugins#26] [Help] [answered]
Can anyone please help me? I'm desperate after I would clean my database and generally my whole site and firstly exported Networks and Merchants. 
After installing my site again and all plugins I Imported them again but I only se 1 Unnamed Network with no Merchants in it so I tried to reinstall everything again because I suspected that it could have been saved in the database but no because everything had been refreshed. I have emptied the browser cache with the same result and then tried another browser and still the same problem.

I hope that any of you have a good idé.

Thank you.

**Comment by fiversen-hofor** [comment, marked as answer]
Arh I have forgotten to update my credit card, sorry for disturbing ;-)

**Comment by stev1972** [comment]
When you posted this you had no active subscription. Without active subscription it's not possible to connect to our database. I noticed you renewed your subscription in the meantime. Can you try again? If you still experience any problems can you try to delete your API cache here: WordPress admin > Datafeedr API > Tools?
